flent/runners.py

```diff
@@ -844,6 +844,59 @@ def find_binary(cls, host, interval, length, ip_version, local_bind=None,
 
         raise RuntimeError("No suitable Iperf binary found.")
 
+class SsRunner(ProcessRunner):
+    """Runner for iterated `ss -t -i -p`. Depends on same partitial output
+    separationa and time stamping as TcRunner."""
+
+    time_re = re.compile(r"^Time: (?P<timestamp>\d+\.\d+)", re.MULTILINE)
+    
+    CWND_IDX = 0
+    RTT_IDX = 1
+    PID_IDX = 2
```

Comment:
This positional approach made the Runner parsing code less convoluted, though I am prepared to align. See your point.